pubspec.yaml

```diff
@@ -3,7 +3,7 @@
 # license that can be found in the LICENSE file.
 
 name: codemirror
-version: 0.5.9+5.48.4
+version: 0.5.10
```

Comment:
perhaps `0.5.10+5.48.4`; `5.48.4` is the JS code mirror version

Reply:
Ah that is what that naming scheme was. We'll start using it for js interop based libraries in devtools as well.